app/org/sagebionetworks/bridge/services/StudyService.java

```diff
@@ -538,42 +562,31 @@ private boolean studyDirectoryHasChanged(Study originalStudy, Study study) {
     }
     
     /**
-     * When the password policy or templates are not included, they are set to some sensible default 
+     * When the password policy or templates are not included, they are set to some sensible defaults.  
      * values. 
      * @param study
      */
     private void setDefaultsIfAbsent(Study study) {
         if (study.getPasswordPolicy() == null) {
             study.setPasswordPolicy(PasswordPolicy.DEFAULT_PASSWORD_POLICY);
         }
-        study.setVerifyEmailTemplate(fillOutTemplate(study.getVerifyEmailTemplate(),
-            defaultEmailVerificationTemplateSubject, defaultEmailVerificationTemplate));
-        study.setResetPasswordTemplate(fillOutTemplate(study.getResetPasswordTemplate(),
-            defaultResetPasswordTemplateSubject, defaultResetPasswordTemplate));
-    }
-
-    /**
-     * Partially resolve variables in the templates before saving on Stormpath. If templates are not provided, 
-     * then the default templates are used, and we assume these are text only, otherwise this method needs to 
-     * change the mime type it sets.
-     * @param template
-     * @param defaultSubject
-     * @param defaultBody
-     * @return
-     */
-    private EmailTemplate fillOutTemplate(EmailTemplate template, String defaultSubject, String defaultBody) {
-        if (template == null) {
-            template = new EmailTemplate(defaultSubject, defaultBody, MimeType.HTML);
+        if (study.getVerifyEmailTemplate() == null) {
+            EmailTemplate template = new EmailTemplate(defaultEmailVerificationTemplateSubject,
+                    defaultEmailVerificationTemplate, MimeType.HTML);
+            study.setVerifyEmailTemplate(template);
         }
-        if (StringUtils.isBlank(template.getSubject())) {
-            template = new EmailTemplate(defaultSubject, template.getBody(), template.getMimeType());
```

Comment:
This change means if we have a template with a subject and no body (or a body and no subject), we replace the whole template instead of filling in the missing attributes. Is this expected?